bcda/client/bluebutton.go

```diff
@@ -123,7 +123,7 @@ func NewBlueButtonClient(config BlueButtonConfig) (*BlueButtonClient, error) {
 	}
 	var timeout int
 	if timeout, err = strconv.Atoi(conf.GetEnv("BB_TIMEOUT_MS")); err != nil {
-		logger.Info("Could not get Blue Button timeout from environment variable; using default value of 500.")
+		logger.Info(errors.Wrap(err, "Could not get Blue Button timeout from environment variable; using default value of 500."))
```

Comment:
why aren't we using logger.Error here?

Reply:
Similar to my comment above — will change this to `Warn`.